readme.md

```diff
@@ -59,6 +59,17 @@ services:
       ES_JAVA_OPTS: "-Xms2g -Xmx2g"
 ```
 
+## Xdebug
+
+Add the following lines to `docker-php-ext-xdebug.ini` in order to use Xdebug.
+
+```
+xdebug.remote_enable = 1
+xdebug.remote_host = {YOU_LOCAL_IP_ADDRESS}
```

Comment:
Is this you local address in the network? If your DHCP lease expires you have to change it?

Reply:
How do I get the correct IP address? Is it the PHP-FPM IP address that docker resolves ?